### PR TITLE
[Display] Add Divider

### PR DESCRIPTION
**Pull request by MarcoPOtard**

### CODE REVIEW

**Comment by MatouTechnologies** [comment]
This pull request has been deployed to Vercel.
<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://matou-technologies-purrfect-ui-storybook-22.vercel.app'>https://matou-technologies-purrfect-ui-storybook-22.vercel.app</a></td>
  </tr>
  <tr>
    <td><strong>🔍 Inspect:</strong></td>
    <td><a href='https://vercel.com/matou-technologies/purrfect-ui/DtmnQfL1iczR8fbiKuPrGYRQTbrp'>https://vercel.com/matou-technologies/purrfect-ui/DtmnQfL1iczR8fbiKuPrGYRQTbrp</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="deploy-preview-storybook"></hidden>

File: src/display/Divider/Divider.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    },
+  },
+
+  render: (args) => (
+    <>
+      <Title>Divider</Title>
+      <StyledDivContainer orientation={args.orientation}>
+        <StyleText>A little text before</StyleText>
+        <Divider {...args} />
+        <StyleText>A little text before</StyleText>
```
Comment (Balkeo) [inline src/display/Divider/Divider.stories.tsx:75]:
```suggestion
        <StyleText>A little text after</StyleText>
```

File: src/display/Divider/Divider.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const Divider = (props: DividerProps) => {
+  const StyledSeparator = styled(Separator)`
+    border: 0;
+    background-color: ${theme.color.gray};
+    width: ${props.orientation === "vertical" ? "1px" : "100%"};
+    height: ${props.orientation === "vertical" ? "100%" : "1px"};
+    margin: ${props.orientation === "vertical"
+      ? `0 ${sizing(5)}`
+      : `${sizing(5)} 0`};
+  `;
```
Comment (Balkeo) [inline src/display/Divider/Divider.tsx:17]: Move this out of the Divider component

File: src/display/Divider/Divider.stories.tsx (hunk 1 above)
Comment (MarcoPOtard) [reply src/display/Divider/Divider.stories.tsx:75]: corrigé

**Pull request merged**